ui/src/shared/components/Gauge.tsx

```diff
@@ -306,14 +308,34 @@ class Gauge extends Component<Props> {
     ctx.textBaseline = 'middle'
     ctx.textAlign = 'center'
 
-    const valueString = this.toDisplayString(gaugePosition)
+    const valueString = this.valueToString(gaugePosition)
 
     const textY = radius
     const textContent = `${prefix}${valueString}${suffix}`
     ctx.fillText(textContent, 0, textY)
   }
 
-  private toDisplayString(value: number): string {
+  private labelToString(value: number): string {
+    const {decimalPlaces} = this.props
+
+    let valueString
+
+    if (decimalPlaces.isEnforced) {
+      valueString = value.toLocaleString(undefined, {
+        minimumFractionDigits: 0,
+        maximumFractionDigits: decimalPlaces.digits,
+      })
+    } else {
+      valueString = value.toLocaleString(undefined, {
+        minimumFractionDigits: 0,
+        maximumFractionDigits: 20,
```

Comment:
Could this use the constant  mentioned above?

Reply:
one is a string and one is a number but sure.